projects/canopy/src/lib/paginator/paginator.component.html

```diff
@@ -0,0 +1,52 @@
+<lg-select-field class="lg-paginator__page-size" *ngIf="!hidePageSize">
+  <span class="lg-paginator__page-size-label">Items per page</span>
+
+  <select lgSelect [ngModel]="pageSize" (ngModelChange)="handleSelectPageSize($event)">
+    <option *ngFor="let size of pageSizes" [value]="size">{{ size }}</option>
+  </select>
+</lg-select-field>
+
+<span class="lg-paginator__status"
+  >{{ currentPageStart }} - {{ currentPageEnd }} of {{ length }}</span
+>
```

Comment:
Perhaps we could also have a `showPaginatorStatus` input? Not sure about this one, but there could be a scenario where we have a very simple 2-page pagination where this might be clutter. Again bit of a design question too. Perhaps also for accessibility, it should always be readable by the screenreader, regardless if shown visually.

Reply:
Sure, I refrained from putting any options in to control labels etc until there was a design decision but it's definitely worth adding that.